test_orquestra.py

```diff
@@ -0,0 +1,4 @@
+from examples.orquestra.hydrogen_demo.run import main
```

Comment:
As far as I can tell this is a temporary file on this branch?

Reply:
Already removed 👍